content/en/blog/2021/stateful-set-made-easier/index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "StatefulSets Made Easier With Istio 1.10"
+description: Learn how to easily deploy StatefulSets with Istio 1.10.
+publishdate: 2021-05-18
+attribution: "Lin Sun (Solo.io), Christian Posta (Solo.io), John Howard (Google), Zhonghu Xu (Huawei)"
+keywords: [statefulset,Istio,networking,localhost,loopback,eth0]
+---
+
+Kubernetes [StatefulSets](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) are commonly used to manage stateful applications. In addition to managing the deployment and scaling of a set of Pods, StatefulSets provide guarantees about the ordering and uniqueness of those Pods. Common applications used with StatefulSets include ZooKeeper, Cassandra, Elasticsearch, Redis and NiFi.
```

Comment:
I will also add that Pod usually has backticks as well, at least when capitalized.